oak_functions/client/rust/src/main.rs

```diff
@@ -19,10 +19,13 @@
 
 use anyhow::Context;
 use clap::Parser;
-use oak_functions_abi::proto::Request;
+use oak_functions_abi::Request;
 use oak_functions_client::Client;
 use regex::Regex;
 
+const TWO_MIB: usize = 2097000;
```

Comment:
That's a really weird constant. It's not 2\*1000^2 (2_000_000) nor 2\*1024^2 (2_097_152). What's going on here?

Reply:
Pre-existing, but fixed, thanks.